awx/main/models/events.py

```diff
@@ -271,34 +271,44 @@ def get_event_display2(self):
 
     def _update_from_event_data(self):
```

Comment:
Same comment as https://github.com/ansible/awx/pull/5618/files#r364842220

You'll notice I've removed the `updated_fields` tracking because this method doesn't actually call `.save()` (it just updates properties of `self`).